pkg/tests/tasks/traffic/common.go

```diff
@@ -0,0 +1,62 @@
+package traffic
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/template"
+
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestreviewV1(t TestHelper, file string) string {
+
+	ns := "bookinfo"
+
+	reviewV1Podname := pod.MatchingSelector("app=reviews,version=v1", ns)(t, oc.DefaultOC).Name
+
+	templateString, err := os.ReadFile("../../../../testdata/resources/html/" + file)
+	if err != nil {
+		t.Fatalf("could not read template file %s: %v", templateString, err)
```

Comment:
```suggestion
		t.Fatalf("could not read template file %s: %v", file, err)
```
`file` contains the file name you want to print for the error message.

Reply:
If, we use always file. will confuse with the code.

We can use this templateString, templateString2, and templateString3. It is easy to read it.

Reply:
If there was an error, that means that `os.ReadFile` did not successfully write the content of the file into `templateString` variable, which means you should not attempt to use the variable (unless you are overwriting it). Trying to use output variables after an error has occurred, unless the function you are calling says otherwise in its documentation, could produce undefined behavior.

Also putting `templateString` there does not cause it to print the name of the variable (as in "could not read template file templateString: "), it will actually print the content of the variable into the format string (where you see the `%s`). So, assuming there was no error, the message would be:
```
could not read template file <!DOCTYPE html>
<html>
  <head>
...
[the rest of the template file]
```

If you use the `file` variable there instead, it will print the name of the file it was trying to read (e.g. "could not read template file productpage-test-user-v2-rating-unavailable.html").